integration-test/rollingupdate-kubernetes/src/main/scala/akka/cluster/bootstrap/PodDeletionCostDemoApp.scala

```diff
@@ -23,7 +23,11 @@ object PodDeletionCostDemoApp extends App {
 
   AkkaManagement(system).start()
   ClusterBootstrap(system).start()
-  PodDeletionCost(system).start()
+  if (Option(System.getenv("USE_APP_VERSION_REVISION")).nonEmpty) {
+    AppVersionRevision(system).start()
+  } else {
+    PodDeletionCost(system).start()
+  }
```

Comment:
> Question: Should extract into own class? Currently the integration test runs off PodDeletionCostDemoApp.scala

I think this is fine. Would probably be ok without the USE_APP_VERSION_REVISION toggle as well. Aways start both.

Reply:
Right! I did not want to create the CR needed for the PodDeletionCost, but now I realize that it will just do its thing in the background and no mess with what I am testing.